src/Symfony/Component/Intl/Tests/Locale/LocaleTest.php

```diff
@@ -153,6 +153,8 @@ public function testSetDefault()
     public function testSetDefaultAcceptsEn()
     {
         $this->call('setDefault', 'en');
+
+        $this->assertSame('en', $this->call('getDefault'));
```

Comment:
This change is already in the `2.8` branch because of #22379. It looks like I missed it for the `2.7` branch in #22066.